pages/tutorials/[slug].js

```diff
@@ -1,49 +1,69 @@
 import fs from 'fs'
 import path from 'path'
 import matter from 'gray-matter'
-import { marked } from 'marked'
 import styled from 'styled-components'
 import DynNavbar from '../../components/DynNavbar'
 import Footer from '../../components/Footer'
+import { serialize } from 'next-mdx-remote/serialize'
+import { MDXRemote } from 'next-mdx-remote'
+import remarkMath from 'remark-math'
+import rehypeKatex from 'rehype-katex'
+import rehypeHighlight from 'rehype-highlight'
+import Head from 'next/head'
+
 
 
 const Hero = styled.div`
   display: flex;
-  justify-content: center;
+  justify-content: left;
   background: #fff;
-  align-items: center;
+  align-items: left;
   padding-left: 10%;
   padding-right: 10%;
   padding-top: 20px;
   padding-bottom: 20px;
 `
 
+
+
+
 export default function PostPage({
-  frontmatter: { title, cover_image },
-  slug,
-  content,
+  mdxSource,
 }) {
+
+
   return (
     <>
+    <Head>
+      <link
+        rel="stylesheet"
+        href="https://cdn.jsdelivr.net/npm/katex@0.13.11/dist/katex.min.css"
+        integrity="sha384-Um5gpz1odJg5Z4HAmzPtgZKdTBHZdw8S29IecapCSB31ligYPhHQZMIlWLYQGVoc"
+        crossOrigin="anonymous"
+      />
+      <link 
+        rel="stylesheet" 
+        href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.7.0/styles/shades-of-purple.min.css">
+      </link>
```

Comment:
Same here: this is yet another CDN, so the site can be broken because of two more providers.
If possible, get the CSS from the `highlight.js` npm package.

Reply:
Thank you. 
This solves another problem: next suggested me to avoid the use of `<Head />`, preferring a more dynamic construction of the decoration procedure. 
I wanted to do that, but the problem vanished by uploading the `*.css` files directly from the modules folder. 

Only a question: the `node_modules` are not involved in the git track of the files, of course. This problem (to upload the `*.css`) is solved by GitHub when the workflow  install, compile and build the site?

I think that also @Edoardo-Pedicillo can answer to my  doubt.

Reply:
`yarn install` (or any other node package manager) creates the `node_modules` installing the deps.
If the lock-file is available, it will be used, resolving the exact same packages you have locally (if those are compliant with the lock-file, of course).

`node_modules` is essentially the equivalent of a Python environment :)

Reply:
Ok, thank you very much.